recipes/chemnlp/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "chemnlp" %}
+{% set version = "2023.3.4" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/chemnlp-{{ version }}.tar.gz
+  sha256: 071b68f37ea3f121ea7b48bd3669a9be802e4cc624185a13de0f659e459ac6ba 
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - run_chemnlp.py=chemnlp.utils.run_chemnlp:cli
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - numpy >=1.22.0
+    - scipy >=1.6.3
+    - jarvis-tools >=2021.07.19
+    - chemdataextractor >=1.3.0
+    - matplotlib-base >=3.4.1
+    - pandas ==1.2.4
```

Comment:
Is this tight pinning of pandas necessary? I don't see it in upstream's `setup.py`?

Reply:
Just modified the file.